templates/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="col">
+  <p>IATI Query Builder is free software licenced under the GNU General Public License.<br/>
+  <ul>
+    <li><a href="https://github.com/IATI/IATI-Query-Builder" target="_blank" rel="noopener noreferrer nofollow">IATI Query Builder on GitHub</a></li>
+    <li>Please report problems to our <a href="https://github.com/IATI/IATI-Query-Builder/issues" target="_blank" rel="noopener noreferrer nofollow">issues list</a>.</li>
+    <li>Datastore documentation: <a href="http://datastore.iatistandard.org/" target="_blank" rel="noopener noreferrer">http://datastore.iatistandard.org</a></li>
+  </ul>
+  </p>
```

Comment:
The HTML spec says the paragraph close tag goes before the list https://stackoverflow.com/a/10601411